modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-gcp-cloud-file-delete-instances_{context}"]
+= Deleting GCP Filestore instances
+
+Typically, if you destroy a cluster, the {product-title} installer deletes all of the cloud resources that belong to that cluster. However, when a cluster is destroyed, Google Compute Platform (GCP) Filestore instances are not automatically deleted, so you must manually delete all persistent volume claims (PVCs) that use the Filestore storage class before destroying the cluster.
+
+.Prerequisites
+* Access to Google Cloud console
+
+.Procedure
+To delete an instance:
+
+. In the Google Cloud console, go to the *Filestore Instances* page.
+
+. To to open the instance details page, click the *Filestore instance ID*.
```

Comment:
Multiple of "to"

Reply:
Fixed by removing and replacing this content.